src/main.rs

```diff
@@ -46,6 +67,12 @@ fn main() {
                 .required(true)
                 .help("Minimum number of shares required to reconstruct the secret (1 <= t <= n)"),
         )
+        .arg(
+            Arg::with_name("format-type")
+                .long("format-type")
+                .takes_value(true)
+                .help("Field value formatting type possible values include: [hex|bip39] [default: hex]")
```

Comment:
Rather than documenting the possible values and the default, you can use the `possible_values()` and  `default_value()` functions provided by clap. See the `--type` argument.